op-node/withdrawals/utils.go

```diff
@@ -264,23 +267,17 @@ func ParseWithdrawalInitiated(receipt *types.Receipt) (*bindings.L2ToL1MessagePa
 	if err != nil {
```

Comment:
two lines above needs a doc comment (though it was probably me that forgot to write it in the first place).

Reply:
Done.